### PR TITLE
DTGB-417: Fixed some things for use in D8.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: source/sass/11-base/contextual-links/_contextual-links.scss [hunk 1]
```diff
@@ -5,8 +5,12 @@
   .trigger,
   .trigger:focus,
   .trigger:hover {
-    background: $color-white url("../../../../contrib/gent_base/source/img/pencil.svg") no-repeat center center !important;
-    background-size: .75rem !important;
+    @include theme('background-color', 'color-secondary', 'contextual-link-background');
+
+    background-image: url("../../../../contrib/gent_base/source/img/pencil.svg");
+    background-repeat:no-repeat;
```
Comment (Jeroen005) [inline source/sass/11-base/contextual-links/_contextual-links.scss]: space

File: styleguide/components/11-base/fonts/_icons_template.template [hunk 2]
```diff
@@ -31,7 +31,7 @@ $id: random(1000);
   text-transform: none;
 }
 
-[class*='<%= cssClass%>'],
+[class^='<%= cssClass%>'],
```
Comment (Jeroen005) [inline styleguide/components/11-base/fonts/_icons_template.template]: bewust? je moet hem nu vooraan  in het attr gebruiken :)

File: templates/core/misc/status-messages.html.twig [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <i class="icon-checkmark" aria-hidden="true"></i>
+      {% endif %}
+      {% if type == 'warning' %}
+        <i class="icon-exclamation" aria-hidden="true"></i>
+      {% endif %}
+      {% if type == 'error' %}
+        <i class="icon-cross" aria-hidden="true"></i>
+      {% endif %}
+      {% if messages|length > 1 %}
+        <p>
```
Comment (Jeroen005) [inline templates/core/misc/status-messages.html.twig]: Is deze p noodzakelijk?

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: source/sass/11-base/contextual-links/_contextual-links.scss (hunk 1 above)
Comment (meirege) [reply source/sass/11-base/contextual-links/_contextual-links.scss]: Fixed.

File: styleguide/components/11-base/fonts/_icons_template.template (hunk 2 above)
Comment (meirege) [reply styleguide/components/11-base/fonts/_icons_template.template]: Fixed.

File: templates/core/misc/status-messages.html.twig (hunk 3 above)
Comment (meirege) [reply templates/core/misc/status-messages.html.twig]: Nope, I deleted it.

**Pull request merged**